source/user-manual/wazuh-indexer/index-life-management.rst

```diff
@@ -1,11 +1,107 @@
 .. Copyright (C) 2015, Wazuh, Inc.
 
 .. meta::
-   :description: Learn how to define index retention policies in this section of the documentation.
+   :description: Learn how to define index management policies in this section of the documentation.
 
 Index life management
 =====================
 
+Using Index State Management policies will allow you to define the lifecycle for an index from the ingestion to the deletion stage, optimizing the cluster's performance and establishing a retention policy for your data.
+
+What is an ISM policy?
+----------------------
+
+Extracted from `OpenSearch's documentation
+<https://opensearch.org/docs/latest/im-plugin/ism/index>`_: 
+
+|   Index State Management (ISM) is a plugin that lets you automate these periodic, administrative operations by triggering them based on changes in the index age, index size, or number of documents. Using the ISM plugin, you can define policies that automatically handle index rollovers or deletions to fit your use case.
+
+Index rollover
+---------------
+
+Since 4.8.0, Wazuh uses a ISM policy to automatically rollover indices based on the number of documents, index size, or index age.
+
+    .. code-block:: json
+        :emphasize-lines: 12,13,14
+
+        {
+            "policy": {
+                "description": "Wazuh rollover and alias policy",
+                "default_state": "active",
+                "states": [
+                {
+                    "name": "active",
+                    "actions": [
+                    {
+                        "rollover": {
+                            "min_primary_shard_size": "25gb",
+                            "min_index_age": "7d",
+                            "min_doc_count": "${rollover_min_docs}"
+                        }
+                    }
+                    ]
+                }
+                ],
+                "ism_template": {
+                    "index_patterns": ["wazuh-alerts*", "wazuh-archives*", "-wazuh-alerts-4.x-sample*"],
+                    "priority": "50"
+                }
+            }
+        }
+
+
+This policy will rotate the ``wazuh-alerts`` and ``wazuh-archives`` indices when any of the following conditions are met:
+
+    * The index is older than 7 days.
+    * An index's primary shard is larger than 25 GB.
+    * The index has more than 2,100,000 documents per shard.
+
+   .. note::
+      
+      The ``rollover_min_docs`` variable is computed based on the number of shards and replicas of the index. There is a limit of 2^31 documents per shard, so the default value is 2,100,000 documents per shard, a bit below the maximum.
```

Comment:
`2,100,000` is ~1022 times smaller than `2^31 = 2,147,483,648`, not "a bit below". What are the actual figures here? If for example the limit were `2^21 = 2,097,152` the default value would exceed it a bit so maybe we should consider a lower default value such as `2,000,000`.

Reply:
This is a typo. Actual value should have been 2,100000,000. Anyway, we are finally removing this setting as we thought it was confusing, and replacing it with the final value instead: `min_doc_count`.